mvi-presenter/src/main/java/pl/valueadd/mvi/presenter/BaseMviPresenter.kt

```diff
@@ -223,7 +231,7 @@ abstract class BaseMviPresenter<VS : IBaseViewState, PS : IBasePartialState, VI
                 .doOnNext { currentState = it }
                 .subscribe(
                     viewStateBehaviorSubject::onNext,
-                    viewStateBehaviorSubject::onError
+                    ::onError
```

Comment:
When I use viewStateBehaviorSubject, I expect that the observable notfies me about an error but in that implementation this will not happen.

So I suggest to rollback that change and simply add:
`doOnError(this::onError)`